prime-router/docs/authz/authz-transition-plan.md

```diff
@@ -5,35 +5,78 @@ This document will detail the steps needed to successfully transition from the c
 * Implement profile groups (API > ReportStream API > Claims > sender_name)
 * API changes for new auth flow
   * Need to decide if cutting over or maintaining both APIs - see analysis in Questions section 
+* Should sender groups be stored as Okta groups or solely as profile attributes? 
+  * Utilizing groups permits onboarding via Okta admin console, but the group memberships are redundant as it is only
+    the profile attributes that are actively used during the authz process
+  * Storing solely as profile attributes would require the sender management APIs to be built before transition to the
+    auth service can occur
+  * We can change this strategy at any time
 * Create test workflow in staging representative of production environment
   * Update workflow documentation to create new sender
+* Add auth/submissions endpoints to deployment and CI processes
+  * auth and submissions projects currently ignored
+  * end2end testing for endpoints should be created
```

Comment:
Not sure if this needs to be called out here but the end2end_up test will have to be adjusted to work with the new auth. It hits the submission and history endpoints.

Edit: Looks like this partially depends on the question of if we will have both auth schemes running at once or not. I see there's also a comment about making a dupe of it for the microservices 👍 🚀